.github/workflows/main.yml

```diff
@@ -1,8 +1,7 @@
-# evaluating GitHub actions for CI, disconsider failures when evaluating PRs
+# evaluating GitHub actions for CI, disregard failures when evaluating PRs
 #
 # this is still missing:
 # - deploy
```

Comment:
FYI: https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ / https://github.com/marketplace/actions/pypi-publish

Reply:
Thanks for the links!

Reply:
Feel free to ping me for help/review. I'm kinda developing my own framework for GH Apps&Actions + I authored that guide & action...

Reply:
Sure, will do! I don't expect many problems, I've already setup deployment for a number of projects (including `pytest-mock`), but you never know!